libraries/render/src/render/ResampleTask.cpp

```diff
@@ -164,7 +169,7 @@ void UpsampleToBlitFramebuffer::run(const RenderContextPointer& renderContext, c
             batch.setViewportTransform(viewport);
             batch.setProjectionTransform(glm::mat4());
             batch.resetViewTransform();
-            batch.setPipeline(_pipeline);
+            batch.setPipeline(pipeline);
```

Comment:
```suggestion
            batch.setPipeline(args->_renderMode == RenderArgs::MIRROR_RENDER_MODE ? _mirrorPipeline : _pipeline);
```
(this avoids the extra copy when you create `pipeline` above